src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -1061,4 +1061,18 @@ public void testModuleNameInMultiThreadMode() throws Exception {
         }
     }
 
+    /**
+     * This test verifies nothing. It is a workaround for the Jacoco limitations.
```

Comment:
@pbludov My original comment wasn't for removing the verify in the method.
I also mentioned that the reason for the mocking should be the first sentence.

Why can't we verify some type of result?
Can we atleast verify `System.exit` was called?

Reply:
done.